src/brevitas_examples/imagenet_classification/ptq/ptq_common.py

```diff
@@ -557,5 +557,8 @@ def check_positive_int(*args):
     We check that every inputted value is positive, and an integer.
     """
     for arg in args:
+        if not arg:
```

Comment:
Redundant to PR: https://github.com/Xilinx/brevitas/pull/934

Included here to make the tests pass.

Reply:
The PR has been merged so you can just rebase now